src/simplicial-hott/09-yoneda.rzk.md

```diff
@@ -400,6 +400,79 @@ Naturality in `#!rzk C` is not automatic but can be proven easily:
         is-natural-in-family-yon-once-pointwise
           A is-segal-A a C D is-covariant-C is-covariant-D ψ u x)
 ```
+We prove the Yoneda embedding.
+
+```rzk tittle="Yoneda embedding, Definition 9.3 RS17 "
+#def yoneda-embedding
+  ( A : U)
+  ( is-segal-A : is-segal A)
+  ( a a' : A)
+  :  hom A a' a → ( (z : A) → hom A a z → hom A a' z)
+  :=
+     yon
+     ( A )
+     ( is-segal-A)
+     ( a )
+     ( \ x → hom A a' x )
+     ( is-covariant-representable-is-segal
+        ( A)
+        ( is-segal-A)
+        ( a'))
+```
+```rzk
+#def yoneda-comp uses (funext)
+  ( A : U)
+  ( is-segal-A : is-segal A)
+  ( a a' : A)
+  ( φ : ( x : A) → ( hom A a x → hom A a' x) )
+  : ( ( yoneda-embedding A is-segal-A a a' ) ( ( evid A a ( hom A a')) φ)) = φ
+  :=
+     yon-evid
+      ( A)
+      ( is-segal-A)
+      ( a)
+      ( hom A a')
+      ( is-covariant-representable-is-segal
+        ( A)
+        ( is-segal-A)
+        ( a')) φ
+```
+```rzk
+#def is-composition
```

Comment:
I don't love the name `is-composition`. The map you're defining is precomposition with `evid A a ( hom A a')) φ`. So maybe something like `precomposition-evid-is-segal`?